src/metaswitch/ellis/prov_tools/display_user.py

```diff
@@ -45,7 +45,7 @@
 def main():
     parser = argparse.ArgumentParser(description="Display user")
     parser.add_argument("-k", "--keep-going", action="store_true", dest="keep_going", help="keep going on errors")
-    parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="suppress errors when ignoring them")
+    parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="suppress errors while ignoring them")
```

Comment:
I'm not sure either of these versions are particularly useful - how about just "suppress non-critical errors"